pavelib/assets.py

```diff
@@ -384,14 +384,36 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
     """
     Compile CoffeeScript to JavaScript.
+
+    Arguments:
+        coffee_files: (optional) packed list of files (passed as individual args) to be converted from .coffee to .js
+    Example:
+        compile_coffeescript("file1.js", "file2.js", "file3.js")
+
+    If no args are passed, the Coffeescript binary will instead be passed a bash substitution that,
+    when expanded, lists all Coffeescript files in the project. All Coffeescript in the project will be compiled.
     """
-    if not files:
-        files = ["`{}`".format(coffeescript_files())]
+
+    coffee_find_cmd = coffeescript_files()
```

Comment:
Oh, sorry, I misread. I thought `cmd` executed the command, but I see that it doesn't (you have to pass it to `sh` to execute it).